--- ActionMap.addActionEntries.pre
+++ ActionMap.addActionEntries.post
@@ -1,11 +1,11 @@
     pub usingnamespace struct {
-        extern fn g_action_map_add_action_entries(ActionMap, [*]Gio.ActionEntry, i32, ?*anyopaque) void;
+        extern fn g_action_map_add_action_entries(ActionMap, [*]Gio.ActionEntry.cType(), i32, ?*anyopaque) void;
         /// @self: Self
-        /// @entries: [*]Gio.ActionEntry
+        /// @entries: [*]Gio.ActionEntry.cType()
         /// @n_entries: i32
         /// @user_data: (nullable) ?*anyopaque
         /// Return: void
-        pub fn addActionEntries(self: ActionMap, arg_entries: []Gio.ActionEntry, arg_user_data: ?*anyopaque) void {
+        pub fn addActionEntries(self: ActionMap, arg_entries: []Gio.ActionEntry.cType(), arg_user_data: ?*anyopaque) void {
             var entries = arg_entries.ptr;
             var n_entries: i32 = @intCast(i32, arg_entries.len);
             _ = g_action_map_add_action_entries(self, entries, n_entries, if (arg_user_data) |some| some else null);
