--- TextBuffer.getEndIter.pre
+++ TextBuffer.getEndIter.post
@@ -5,7 +5,7 @@
         /// Return: void
         pub fn getEndIter(self: TextBuffer, arg_iter: Gtk.TextIter) Gtk.TextIter {
             var iter_mut = arg_iter;
-            _ = gtk_text_buffer_get_end_iter(self, &iter_mut);
+            _ = gtk_text_buffer_get_end_iter(self, iter_mut);
             return iter_mut;
         }
     };
