--- TextIter.forwardSearch.pre
+++ TextIter.forwardSearch.post
@@ -14,7 +14,7 @@
         } {
             var match_start_mut = arg_match_start;
             var match_end_mut = arg_match_end;
-            var ret = gtk_text_iter_forward_search(self, arg_str, arg_flags, &match_start_mut, &match_end_mut, arg_limit);
+            var ret = gtk_text_iter_forward_search(self, arg_str, arg_flags, match_start_mut, match_end_mut, arg_limit);
             return .{ .ret = ret, .match_start = match_start_mut, .match_end = match_end_mut };
         }
     };
\ 文件尾没有换行符
