--- TextBuffer.getStartIter.pre
+++ TextBuffer.getStartIter.post
@@ -5,7 +5,7 @@
         /// Return: void
         pub fn getStartIter(self: TextBuffer, arg_iter: Gtk.TextIter) Gtk.TextIter {
             var iter_mut = arg_iter;
-            _ = gtk_text_buffer_get_start_iter(self, &iter_mut);
+            _ = gtk_text_buffer_get_start_iter(self, iter_mut);
             return iter_mut;
         }
     };
