--- a/c_linux.zig
+++ b/c_linux.zig
@@ -19456,5 +19456,8 @@
     _: u30,
 };
-pub const struct__PangoGlyphVisAttr = opaque {};
+pub const struct__PangoGlyphVisAttr = packed struct(u32) {
+    is_cluster_start: bool,
+    is_color: bool,
+    _: u30,
+};
 pub const PangoGlyphVisAttr = struct__PangoGlyphVisAttr;
 pub const struct__PangoGlyphInfo = extern struct {
@@ -28908,5 +28911,5 @@
 pub extern fn gsk_text_node_has_color_glyphs(node: ?*const GskRenderNode) gboolean;
 pub extern fn gsk_text_node_get_num_glyphs(node: ?*const GskRenderNode) guint;
-pub extern fn gsk_text_node_get_glyphs(node: ?*const GskRenderNode, n_glyphs: [*c]guint) ?*const PangoGlyphInfo;
+pub extern fn gsk_text_node_get_glyphs(node: ?*const GskRenderNode, n_glyphs: [*c]guint) [*]const PangoGlyphInfo;
 pub extern fn gsk_text_node_get_color(node: ?*const GskRenderNode) [*c]const GdkRGBA;
 pub extern fn gsk_text_node_get_offset(node: ?*const GskRenderNode) [*c]const graphene_point_t;
